deploy/docker/entrypoint.sh

```diff
@@ -126,10 +126,8 @@ init_replica_set() {
   if [[ $isUriLocal -gt 0 ]]; then
     # Check mongodb cloud Replica Set
     echo "Checking Replica Set of external MongoDB"
-    responseStatus=$(mongo "$APPSMITH_MONGODB_URI" --eval "rs.status()" | grep ok | xargs)
-    okString="ok : 1"
 
-    if [[ $responseStatus == *$okString* ]]; then
+    if $(appsmithctl check_replica_set); then
```

Comment:
I'm not sure this would work. For this to work, the command `appsmitctl check_replica_set` should _print_ `true` or `false`. But from the implementation it doesn't seem to do that.

Instead the command exits with 0 or 1 status code. To make use of that, this should be written without interpolation, like this:

```suggestion
    if appsmithctl check_replica_set; then
```

Reply:
thanks for the suggestion. We updated it.